### PR TITLE
Contact form added

### PR DESCRIPTION
**Pull request by nurulhassan**

### CODE REVIEW

File: src/app/contact/contact.component.html [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+        <h5 class="sub_head">Solve your queries here</h5>
+        <form>
+          <div class="form-group">
+            <label for="name" class="bmd-label-floating">Name</label>
+            <input type="text" class="form-control" id="">
+            <label for="email" class="bmd-label-floating">Email address</label>
+            <input type="email" class="form-control" id="">
+            <label for="subject" class="bmd-label-floating">Subject</label>
+            <input type="text" class="form-control" id="">
+            <label for="email" class="bmd-label-floating">Description</label>
```
Comment (laveesh) [inline src/app/contact/contact.component.html]: Naming is not proper

File: src/app/contact/contact.component.html [hunk 2]
```diff
@@ -1,3 +1,61 @@
...
+          </div>  
+          <button type="submit" class="btn btn-primary btn-raised align-center ">Submit</button>
+        </form>
+      </div>
+      <div class="col-md-4 offset-md-1 contact_form card backcard_inner">
+        
+        <h5 class="sub_head">Contact Information</h5>
+        <ul class="list-unstyled">
+          <li class="email_head">
+            <i class="fa fa-envelope" aria-hidden="true"></i> <span> Send us an E-mail :</span><a href="mailto:nurulhassan.it20@jecrc.ac.in" class="text-center pt-8">
```
Comment (laveesh) [inline src/app/contact/contact.component.html:30]: update email as sample@edupro.com

File: src/app/contact/contact.component.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
+$contactback_img:url("../../assets/images/contact_background.jpg");
+$background-color:  #f6f4f8;
```
Comment (laveesh) [inline src/app/contact/contact.component.scss]: use single style write name 
recommended: camelCase

File: src/app/contact/contact.component.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+$font-color:#9e519f;
```
Comment (laveesh) [inline src/app/contact/contact.component.scss:3]: change style

**Review by laveesh — CHANGES_REQUESTED** [review]

**Comment by laveesh** [comment]
Please use single writing style
Please ref to: https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849

File: src/app/contact/contact.component.html (hunk 1 above)
Comment (nurulhassan) [reply src/app/contact/contact.component.html]: naming fixed

**Review by laveesh — APPROVED** [review]

**Pull request merged**